lib/Backend/Lower.cpp

```diff
@@ -8313,7 +8320,7 @@ Lowerer::LowerLdElemI(IR::Instr * instr, IR::JnHelperMethod helperMethod, bool i
 
     src1->Free(this->m_func);
 
-    m_lowererMD.ChangeToHelperCall(instr, helperMethod, nullptr, nullptr, nullptr, isHelper);
+    m_lowererMD.ChangeToHelperCall(instr, helperMethod, labelBailOut, nullptr, nullptr, isHelper);
```

Comment:
Done.

---
In reply to: [67767463](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67767463) [](ancestors = 67767463)